examples/hovercard-shadow-dom/test-browser.ts

```diff
@@ -0,0 +1,30 @@
+import type { Page } from "@playwright/test";
+import { expect, test } from "@playwright/test";
+
+const getAnchor = (page: Page) =>
+  page.getByRole("link", { name: "@ariakitjs" });
+const getHovercard = (page: Page) =>
+  page.getByRole("dialog", { name: "Ariakit" });
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/previews/hovercard-shadow-dom", {
+    waitUntil: "networkidle",
+  });
+});
+
+test.describe.configure({ retries: process.env.CI ? 2 : 1 });
```

Comment:
I suppose this is coming from another test, right? Did you notice any flakiness on CI? If not, I believe we can remove this.

Reply:
Yeah, I copy this from the `hovercard example` but I did not experience any flakiness, so I'm gonna remove it for now.